apps/aetx/src/aetx_sign.erl

```diff
@@ -57,18 +60,24 @@ new(Tx, Signatures) ->
 %% @doc Given a transaction Tx, a private key or list of keys,
 %% return the cryptographically signed transaction using the default crypto
 %% parameters.
--spec sign(aetx:tx(), list(binary()) | binary()) -> signed_tx().
+-spec sign(aetx:tx()|binary(), list(binary()) | binary()) -> signed_tx() | tuple().
```

Comment:
I don't like this spec-change... This function is a signing a transaction, not a general binary.

Reply:
I will fix this. I can move signing of blocks into `aec_blocks` and the param will be a block, not a binary.